### ISSUE TITLE
travis installation may timeout due to scipy

### ISSUE THREAD

**Issue by johnyf** [tulip-control/polytope#22]
For example, a Travis build can [take 10 min](https://travis-ci.org/tulip-control/polytope/builds/99945161).
Support for a pre-installed `scipy` is discussed in [travis-ci #2650](https://github.com/travis-ci/travis-ci/issues/2650). This issue is a note that slow builds are normal, and a reminder to install `numpy` and `scipy` with `apt` and use system-wide site packages, until Travis offers a pre-installed `scipy`.

**Comment by slivingston** [comment]
This is relevant to a [comment](https://github.com/tulip-control/tulip-control/issues/131#issuecomment-150703272) on tulip-control/tulip-control#131

**Comment by johnyf** [comment]
I tried using system site packages and installing `python-scipy`, in addition to `python-numpy`. Whereas this delivers `numpy == 1.6.1` (`numpy >= 1.6` required in `install_requires`, as of 4b3d1c3d28c344285557b4bbdff65aedb88a841a), it gets only `scipy == 0.9.0` (`scipy >= 0.16` required).

The `scipy >= 0.16` requirement (introduced in bd1b30cb1915f875ccc25c790929a83e3e429127) cannot be relaxed to `0.9.0` (2011), because `scipy.optimize.linprog` appears to [have been introduced later](https://github.com/scipy/scipy/commit/a136c152d142b196efe3a5e7db568ad063af98c4) (2013).
So, `pip install scipy` cannot be avoided.

I used [`travis_wait 30 pip install scipy`](http://stackoverflow.com/questions/28746046/how-can-i-install-something-on-travis-ci-without-a-timeout) (minutes).

In the future, a descent solution would be to build a Linux wheel (only OS X wheels are available for `scipy` on PyPI), and place it somewhere convenient, so that we `pip install` that on Travis.
